test/PlaygroundTransform/for_crash.swift

```diff
@@ -1,7 +1,13 @@
 // RUN: %empty-directory(%t)
 // RUN: cp %s %t/main.swift
-// RUN: not %target-swift-frontend -typecheck -playground %t/main.swift
-// RUN: not %target-swift-frontend -typecheck -playground -Xfrontend -pc-macro %t/main.swift
+
+// -playground
+// RUN: not %target-swift-frontend -typecheck -swift-version 5 -playground %t/main.swift
+// RUN: not %target-swift-frontend -typecheck -swift-version 6 -playground %t/main.swift
+
+// -pc-macro -playground
+// RUN: not %target-swift-frontend -typecheck -swift-version 5 -playground -Xfrontend -pc-macro %t/main.swift
```

Comment:
Nit: you might consider `%target-typecheck-verify-swift` when `-typecheck` is involved and I think `-Xfrontend` is redundant since this is already a frontend invocation that is being invoked here.

Reply:
Good tip, thanks. Will update the test.